_docs/15c-permissions-example.md

```diff
@@ -210,3 +254,44 @@ export const SERVICE_CONFIG: Object = {
 };
 ```
 
+Other important configuration it's to define on the `main-routing.module.ts` the `canActivateChild` parameter to manage the routes permissions.
+
+`main-routing.module.ts`:
+```javascript
+...
+export const routes: Routes = [
+  {
+    path: '',
+    component: MainComponent,
+    canActivate: [AuthGuardService],
+    canActivateChild: [PermissionsGuardService],
+    children: [
+...
+```
+
+## How to extend the permission service
+
+In case we wanna change the permission service we need to do two steps:
+* Extend the `OntimizeEEPermissionsService` or the `OntimizePermissionsService` classes to your own service class. Example:
+
+```javascript
+...
+@Injectable()
+export class CustomPermissionsService extends OntimizeEEPermissionsService {
+...
```

Comment:
I think you can write a small example by overriding the loadPermission and configureService methods